blocks/components/editable/index.js

```diff
@@ -266,10 +268,13 @@ export default class Editable extends wp.element.Component {
 			this.focus();
 		}
 
+		// The savedContent var allows us to avoid updating the content right after an onChange call
```

Comment:
I'm not sure I follow this. Why does the value of `this.props.value` differ from `prevProps.value` after the `onChange`? And if it does, why do we not want this to be reflected in the editable?